app/lib/fake_data/api_response.rb

```diff
@@ -0,0 +1,27 @@
+require "ffaker"
+
+module FakeData::ApiResponse
```

Comment:
i wanted to make this more generic so we could use it for different types of responses and use the same json_api wrapper